source/manual/related-links-on-govuk.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Related links on GOV.UK
+parent: "/manual.html"
+layout: manual_layout
+type: learn
+section: Related links
+owner_slack: "#govuk-developers"
+last_reviewed_on: 2019-07-22
+review_in: 6 months
+---
+
+This is technical documentation for the [GOV.UK](https://www.gov.uk/) team in the [Government
```

Comment:
Everything in this repository is tech docs for GOV.UK

Reply:
Fair. I was using another doc as a template and it had this, which means it is also incorrect. I'll remove 👍